definitions/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cleanup-workspace
+spec:
+  params:
+    - name: subdirectory
+      type: string
+      description: The directory to remove within the workspace
+  workspaces:
+    - name: input
+      description: Workspace where the directory to cleanup exists
+  steps:
+    - name: cleanup
+      image: quay.io/hacbs-release/release-utils
+      script: |
+        #!/usr/bin/env sh
+        set -eux
+
+        if [ -z "$(params.subdirectory)" ] ; then
+            echo "The empty string is not a valid subdirectory"
+            exit 1
```

Comment:
What about not breaking here and just show a warning like you do?

Reply:
Why would we want to try to let `rm -rf ${mounted-workspace}` execute?

Reply:
I meant to change it to `exit 0` with just a warning.